projects/efg-training/resources/test/app-subnets.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_subnet" "efg_app_1" {
+    vpc_id            = "${aws_vpc.vpc.id}"
```

Comment:
The VPC is created in https://github.com/alphagov/govuk-terraform-provisioning/pull/36/files#diff-bceb2970582adbc31fcc9e7dbb79d1a6R2 so it should use a distinct one.